qubes/ext/core_features.py

```diff
@@ -104,7 +104,90 @@ async def qubes_features_request(self, vm, event, untrusted_features):
             untrusted_value = untrusted_features["qubes-agent-version"]
             if _version_re.fullmatch(untrusted_value):
                 vm.features["qubes-agent-version"] = untrusted_value
+
+        # handle boot mode advertisement
+        old_bootmode_info = {}
+        for feature_key, feature_val in vm.features.items():
+            if feature_key.startswith(
+                "boot-mode.kernelopts."
+            ) or feature_key.startswith("boot-mode.name"):
```

Comment:
```suggestion
            ) or feature_key.startswith("boot-mode.name."):
```

Reply:
or maybe you meant just `boot-mode.name` for the default boot mode name? in that case, it would need to be handled below too

Reply:
I still don't have default boot mode names implemented, that's still left unresolved from the earlier review. I'll leave some rationale behind leaving them out if I can't get them to work with qubes-manager.

Reply:
Alright, I got default boot mode names working! I went with `default` as the name like you suggested.